scc/light_client/light_client.go

```diff
@@ -21,6 +22,9 @@ type LightClient struct {
 type Config struct {
 	Url     *url.URL
 	Genesis scc.Committee
+	// default values try only once per request with 10 seconds timeout
+	Retries uint
+	Timeout time.Duration
```

Comment:
nit: the `Retries` seems to be redundant and would require knowledge about the internal implementation details to know how to balance it against the `Timeout` parameter. Are 50 retries enough to run into a time-out of 15 seconds? Or do I need to set up at least 500 retries for that time frame?

However, I think the ultimate fix for this will be to use a `Context` as discussed, which will replace those anyway. So let's keep this for the `Context` adding PR.

Reply:
https://github.com/0xsoniclabs/sonic-admin/issues/126